salt/top.sls

```diff
@@ -14,11 +14,17 @@ base:
     - flannel
     - docker
     - haproxy
+    - kube-common
+    - kube-proxy
     - kubelet
     - kubeconfig
   'roles:kube-master':
     - match: grain
     - kubernetes-master
+    - kube-apiserver
+    - kube-controller-manager
+    - kube-scheduler
+    - reboot
```

Comment:
Was adding `reboot` intentional? It looks like this role is not idempotent, so it needs to be fixed before it can be ran multiple times safely.